data/tekton-pipelines/windows-efi-installer-pipeline.yaml

```diff
@@ -0,0 +1,248 @@
+      default: n1.large
+    - name: instanceTypeKind
+      type: string
+      description: Kind of VirtualMachineInstancetype object
+      default: VirtualMachineClusterInstancetype
+    - name: preferenceName
+      type: string
+      description: Name of VirtualMachineClusterPreference object
+      default: windows.11
+    - name: virtualMachinePreferenceKind
+      type: string
+      description: Kind of virtualMachinePreference object
+      default: VirtualMachineClusterPreference
+    - default: windows11-autounattend
+      description: Name of the ConfigMap containing the sysprep configuration files (autounattend.xml, etc.). For example windows11-autounattend or windows2022-autounattend. It is possible to provide customize ConfigMaps created by the user too.
+      name: autounattendConfigMapName
+      type: string
+    - default: quay.io/kubevirt/virtio-container-disk:v0.59.0
+      description: Reference to the containerdisk containing the virtio-win drivers ISO.
+      name: virtioContainerDiskName
+      type: string
+    - default: win11
+      description: Name of the base DataVolume which is created. Pre-installed Windows VMs can be created from this DataVolume.
+      name: baseDvName
+      type: string
+    - default: kubevirt-os-images
+      description: Namespace of the base DataVolume which is created.
+      name: baseDvNamespace
+      type: string
+    - default: win11
+      description: Name of Windows ISO datavolume
+      name: isoDVName
+      type: string
+  tasks:
+    - name: import-win-iso
+      params:
+        - name: manifest
+          value: |-
+            apiVersion: cdi.kubevirt.io/v1beta1
+            kind: DataVolume
+            metadata:
+              annotations:
+                "cdi.kubevirt.io/storage.bind.immediate.requested": "true"
+              generateName: $(params.isoDVName)-
+            spec:
+              source:
+                http:
+                  url: $(params.winImageDownloadURL)
+              storage:
+                volumeMode: Filesystem
+                resources:
+                  requests:
+                    storage: 9Gi
+        - name: waitForSuccess
+          value: true
+        - name: allowReplace
+          value: true
+      taskRef:
+        kind: Task
+        name: modify-data-object
+    - name: modify-windows-iso-file
+      params:
+        - name: pvcName
+          value: $(tasks.import-win-iso.results.name)
+      runAfter:
+        - import-win-iso
+        - create-vm-root-disk
+      taskRef:
+        kind: Task
+        name: modify-windows-iso-file
+    - name: create-vm-root-disk
+      taskRef:
+        kind: Task
+        name: modify-data-object
+      params:
+        - name: manifest
+          value: |-
+            apiVersion: cdi.kubevirt.io/v1beta1
+            kind: DataVolume
+            metadata:
+              annotations:
+                "cdi.kubevirt.io/storage.bind.immediate.requested": "true"
```

Comment:
Quotes used here but in L187 not?

Reply:
Good catch, I updated it, but for some unknow reasons, it was reverted back